letsencrypt/scripts/main.py

```diff
@@ -201,27 +215,95 @@ def read_file(filename):
         raise argparse.ArgumentTypeError(exc.strerror)
 
 
-def rollback(installer, checkpoints):
+def rollback(checkpoints):
```

Comment:
Could we please move `rollback` and other bigger chunks of code to `letsencrypt.client` and keep `scripts` as short as possible?